core/tests/unittests/metrics/test_classification_metrics.py

```diff
@@ -192,6 +192,18 @@ def test_log_loss_with_sklearn(gt, probs):
     np.testing.assert_allclose(ag_loss_as_sklearn, sklearn_log_loss)
 
 
+def test_roc_auc_score_with_sklearn():
```

Comment:
please add empty input `[]` test case and one-value input (int+float check): y_true=1, y_score=0.9

Reply:
Added + fixed edge case difference in output (returning NaN instead of exception), thanks for highlighting